ayon_api/entity_hub.py

```diff
@@ -1485,6 +1486,704 @@ def fill_children_ids(self, children_ids):
+        self._state = state
+
+    def get_color(self):
+        """Get color of project status.
+
+        Returns:
+            str: Status color.
+        """
+
+        return self._color
+
+    def set_color(self, color):
+        """Set color of project status.
+
+        Args:
+            color (str): Color in hex format. Example: '#ff0000'.
+        """
+
+        if not isinstance(color, six.string_types):
+            raise TypeError(
+                "Color must be string got '{}'".format(type(color)))
+        color = color.lower()
+        if self.color_regex.fullmatch(color) is None:
+            raise ValueError("Invalid color value '{}'".format(color))
+        self._color = color
+
+    name = property(get_name, set_name)
+    short_name = property(get_short_name, set_short_name)
+    project_statuses = property(get_project_statuses, set_project_statuses)
+    index = property(get_index, set_index)
+    state = property(get_state, set_state)
+    color = property(get_color, set_color)
+    icon = property(get_icon, set_icon)
+
+    def _validate_other_p_statuses(self, other):
+        o_project_statuses = other.project_statuses
+        m_project_statuses = self.project_statuses
+        if o_project_statuses is None and m_project_statuses is None:
+            raise ValueError("Both statuses are not assigned to a project.")
+
+        missing_status = None
+        if o_project_statuses is None:
+            missing_status = other
+        elif m_project_statuses is None:
+            missing_status = self
+        if missing_status is not None:
+            raise ValueError(
+                "Status '{}' is not assigned to a project.".format(
+                    missing_status.name))
+        if m_project_statuses is not o_project_statuses:
+            raise ValueError(
+                "Statuse are assigned to different projects."
+                " Cannot execute move."
+            )
+
+    def move_before(self, other):
+        """Move status before other status.
+
+        Args:
+            other (ProjectStatus): Status to move before.
+        """
+
+        self._validate_other_p_statuses(other)
+        self._project_statuses.set_status_index(self, other.index)
+
+    def move_after(self, other):
+        """Move status after other status.
+
+        Args:
+            other (ProjectStatus): Status to move after.
+        """
+
+        self._validate_other_p_statuses(other)
+        self._project_statuses.set_status_index(self, other.index + 1)
+
+    def to_data(self):
+        """Convert status to data.
+
+        Returns:
+            dict[str, str]: Status data.
+        """
+
+        output = {
+            "name": self.name,
+            "shortName": self.short_name,
+            "state": self.state,
+            "icon": self.icon,
+            "color": self.color,
+        }
+        if self._original_name and self.name != self._original_name:
+            output["original_name"] = self._original_name
```

Comment:
Should this be `originalName`  ?

Reply:
Not by documentation, asking Marting 
![image](https://github.com/ynput/ayon-python-api/assets/43494761/3965d43e-7995-4976-9cea-a8a9691bb00a)

Reply:
Confirmed it is ok.

Reply:
This feels like a foot-gun ready to be triggered @martastain , just saying... hehe